docs/reference/migration/migrate_6_0/scripting.asciidoc

```diff
@@ -33,4 +33,8 @@ The `lang` variable can no longer be specified as part of a request that uses a
 script otherwise an error will occur.  Note that a request using a stored script is
 different from a request that puts a stored script.  The language of the script has
 already been stored as part of the cluster state and an `id` is sufficient to access
-all of the information necessary to execute a stored script.
\ No newline at end of file
+all of the information necessary to execute a stored script.
+
+==== Stored template apis removed
```

Comment:
Whoa nelly, this should definitely read `Stored *search* template APIs removed`. I had a double-take that we were removing index templates when I first read this.